### PR TITLE
Check that tests were run

### PR DESCRIPTION
**Pull request by amunger**
Run a custom test output parser as a gulp task.
Checks for the test-result.xml file that test runs should be outputting.
Displays info about the test run and details about any test failures.

### CODE REVIEW

**Comment by amunger** [comment]
Need to look at why the failed checks aren't writing test-results files, and the empty nopython suite is a separate issue

File: gulpfile.js [hunk 1]
```diff
@@ -57,6 +57,74 @@ gulp.task('validateTranslationFiles', (done) => {
...
+    const data = await fs.promises.readFile('test-results.xml');
+    const parser = require('xml-js');
+    const report = JSON.parse(parser.xml2json(data, { compact: true }));
+
+    let testsRun = 0;
+    let failCount = 0;
+    let skipCount = 0;
+    try {
+        if (report.testsuites) {
+            core.info(JSON.stringify(report.testsuites._attributes));
```
Comment (rchiodo) [inline gulpfile.js:72]: It looked like this might not be there?

**Review by rchiodo — APPROVED** [review]
